prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -51,6 +51,9 @@ elements:
   - name: previous_message_id
     csvFields: [{name: Corrected_result_ID}]
 
+  - name: test_correction_reason
```

Comment:
Is it ok for you to modify the schema?  The last time I did, they told me that schema is not modifiable.  Please verify with @MauriceReeves-usds.